CONTRIBUTING.md

```diff
@@ -2,59 +2,64 @@
+This guide is for external contributors. Check out our README guidelines here for a general overview of the code repository, how to run the app locally, testing, storage, etc. 
 
-#### Test Accounts
-You can create as many accounts as needed in order to test your changes directly from [expensify.cash](https://expensify.cash/). An initial account can be created when logging in for the first time and additional accounts can be invited by entering a valid email or phone in the "Find or start a chat" input then tapping the avatar.
+We recommend reviewing this “Contributing to Expensify.cash” guide and the README before asking questions, as it may be covered within the documentation.
 
 ## Code of Conduct
 This project and everyone participating in it is governed by the Expensify [Code of Conduct](https://github.com/Expensify/Expensify.cash/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [contributors@expensify.com](mailto:contributors@expensify.com).
 
 ## Asking Questions
-The best way to ask questions is to join our #expensify-contributors Slack channel. To request an invite to the channel, just email contributors@expensify.com with the subject "Slack Channel Invite" and we'll send you an invite! Please do not create issues to ask questions.
+If you have any general questions, please ask in the #expensify-contributors Slack channel. To request an invite to the channel, just email contributors@expensify.com with the subject `Slack Channel Invite` and we'll send you an invite!
+
+If you are hired for an Upwork job, please ask any job-specific questions within the GitHub issue or pull request. This will ensure that the person addressing your question has as much context as possible. The Expensify team will not be able to respond to any direct messages or direct tags in Slack. 
 
 ## Reporting Vulnerabilities
 If you've found a vulnerability, please email security@expensify.com with the subject `Vulnerability Report` instead of creating an issue.
 
-## Filing Issues
-If you'd like to create a new issue, please first make sure the issue does not exist in the [issue list](https://github.com/Expensify/Expensify.cash/issues). When creating a new issue, please include all the required information on the issue template.
-
 ## Payment for Contributions
-We are currently managing payment via Upwork. If you'd like to be paid for your contributions, please apply to fix the issue from our [Upwork issue list](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2). Each issue in this repo will also link out to the associated Upwork job.
-
-## Submitting a Pull Request
```

Comment:
Coming from https://expensify.slack.com/archives/C011W8BJ9L6/p1615481353152800?thread_ts=1615345614.094600&cid=C011W8BJ9L6, let's keep this section here

Reply:
Added back the relevant pieces about signed commits, as the other stuff was already included in our draft.